test/test_article.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+from unittest.mock import MagicMock
+from app import Article
+
+class TestArticle(unittest.TestCase):
+    def setUp(self):
+        self.test_db_articles = ['article_1', 'article_2', 'article_3']
+        self.db_articles = Article()
+        self.output = self.db_articles.get()
+        self.output_not_found = self.db_articles.get(4) # a random index but that is not contained in the test_db_articles Array.
+        self.test_error_message = {'error': 'not found'}
```

Comment:
Excellent tests! 
An advice here would be to reduce the setup and define the variables that are only use for a test in the test itself. It would help to understand a bit better what the test is doing. 
For instance `output_not_found` could be defined in `test_get_article_fail`

Reply:
Great! I'll delete test_get_article_fail as it's not needed anymore (test_errorMessage is enough). Thanks for the suggestion about self.output_not_found.